synthtool/protos/metadata_pb2.py

```diff
@@ -3,6 +3,7 @@
 # source: metadata.proto
```

Comment:
If this change is not necessary for your PR, please could you change it in a separate PR?

Reply:
Removed in recent commit.